modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/BaseMvStoragesTest.java

```diff
@@ -52,58 +52,36 @@
 import org.apache.ignite.lang.IgniteBiTuple;
 import org.apache.ignite.lang.IgniteException;
 import org.jetbrains.annotations.Nullable;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
 
 /**
  * Base test for MV storages, contains pojo classes, their descriptor and a marshaller instance.
  */
 public abstract class BaseMvStoragesTest {
     /** Default reflection marshaller factory. */
-    protected static MarshallerFactory marshallerFactory;
+    protected static final MarshallerFactory marshallerFactory = new ReflectionMarshallerFactory();
 
     /** Schema descriptor for tests. */
-    protected static SchemaDescriptor schemaDescriptor;
+    protected static final SchemaDescriptor schemaDescriptor = new SchemaDescriptor(1, new Column[]{
+            new Column("intKey".toUpperCase(Locale.ROOT), NativeTypes.INT32, false),
```

Comment:
Why do you need **.toUpperCase(Locale.ROOT)**?

Reply:
I don't need it, I moved this code from the `BeforeAll` method to the `clinit` part

Reply:
You did change those lines of code) can we do better then?